openhcl/underhill_core/src/dispatch/mod.rs

```diff
@@ -433,11 +434,32 @@ impl LoadedVm {
         deadline: std::time::Instant,
         capabilities_flags: SaveGuestVtl2StateFlags,
     ) -> anyhow::Result<bool> {
+        if let Some(config) = &self.test_configuration {
+            if config == "SERVICING_SAVE_STUCK" {
+                tracing::info!(
+                    "Test configuration SERVICING_SAVE_STUCK is set. Waiting indefinitely."
+                );
+                loop {
+                    std::thread::sleep(Duration::from_secs(1));
```

Comment:
note that using thread::sleep in an async context is not a good idea, as it doesn't just stall this particular async task, but rather, the entire thread (that may / may-not be multiplexing other async tasks on-top of it).

here, and elsewhere, you should use an async-aware sleep mechanism instead